module/VuFind/src/VuFind/ILS/Driver/Aleph.php

```diff
@@ -567,6 +573,27 @@ public function init()
         if (isset($this->config['Catalog']['default_patron_id'])) {
             $this->defaultPatronId = $this->config['Catalog']['default_patron_id'];
         }
+
+        $this->addressMappings = $this->getDefaultAddressMappings();
+
+        if (isset($this->config['AddressMappings'])) {
+            foreach ($this->config['AddressMappings'] as $key => $val) {
+                $this->addressMappings[$key] = $val;
+            }
+        }
+    }
+
+    protected function getDefaultAddressMappings() {
+        return [
+            'barcode' => 'z304-address-5',
+            'fullname' => 'z304-address-1',
```

Comment:
fullname and barcode are not documented values supported by getMyProfile. I wonder if we should add official support for either or both of these; they have been part of this driver for quite some time, but presumably are not getting used anywhere in core code.

Reply:
We parse firstname and lastname from fullname attribute because Aleph does not store them in separate fields.

Reply:
Makes sense, but my question is, would it be better if the VuFind core looked for a fullname field, and displayed that instead of firstname/lastname when present? If you just use that value for processing internal to the ILS driver, it doesn't necessarily need to be exposed through the data returned by the driver... but if using that exposed data would be useful, it's something we could look into supporting. In any case, it certainly doesn't have to change right now, just something I was thinking about when reviewing the code.